diff --git a/Services/Locator/classes/class.ilLocatorGUI.php b/Services/Locator/classes/class.ilLocatorGUI.php
index 6a217549da..05cbe98a56 100755
--- a/Services/Locator/classes/class.ilLocatorGUI.php
+++ b/Services/Locator/classes/class.ilLocatorGUI.php
@@ -141,7 +141,7 @@ class ilLocatorGUI
 
             // add item for each node on path
             foreach ((array) $path as $key => $row) {
-                if (!in_array($row["type"], array("root", "cat", "crs", "fold", "grp", "prg", "lso"))) {
+                if (!in_array($row["type"], array("root", "cat", "crs", "fold", "grp", "prg", "lso", "xcwi"))) {
                     continue;
                 }
                 if ($crs_ref_id > 0 && $row["child"] == $crs_ref_id) {
